src/hooks/contract.js

```diff
@@ -111,6 +114,7 @@ const ContractContextProvider = ({ Contract, children }) => {
       )
         .then(() => {
           dispatchContract({ type: "CREATE_CORGI_SUCCESS" });
+          window.location.reload(true);
```

Comment:
I always have this question: when creating is triggered, the wallet actually redirect to wallet page and then come back. During this process, the create feature will go back to initial state which is false, the create feature never be true. In this case, I use localStorage to help. But if this works, it is much better. @AnaisUrlichs

Reply:
Tbh I am not sure I what is exactly happening; I realized that the /account page is not refreshed after creating a new corgi -- and that's how I resolved it